src/AC/AC2/Config.h

```diff
@@ -0,0 +1,6 @@
+#define FT_EREG_CHANNEL 31
```

Comment:
These need to be changed:
- call them "FUEL_TANK_EREG_ADDR" and so on; don't abbreviate names
- The address values have changed. see updated ereghat branch